AOT support for a simple console app is failing to build

It seems that the publish folders are missing a 'native' path setup that make the build fails when attempting to publish a project marked with <PublishAot> property.
Steps to reproduce are:

Create simple console .NET app in C# with just one line (example):

```csharp
// See https://aka.ms/new-console-template for more information
Console.WriteLine("Hello, World!");
```

Project content would be:

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net7.0</TargetFramework>
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
    <PublishAot>true</PublishAot>
  </PropertyGroup>

</Project>
```

Drop a 'Directory.Build.props' file on the root of the solution with the following content:

```xml
<Project>

  <PropertyGroup>
    <EnlistmentRoot>$(MSBuildThisFileDirectory)</EnlistmentRoot>
    <CentralBuildOutputPath>$(EnlistmentRoot)</CentralBuildOutputPath>
    <CentralBuildOutputFolderPrefix>___</CentralBuildOutputFolderPrefix>
  </PropertyGroup>

  <Sdk Name="Treasure.Build.CentralBuildOutput"  Version="2.0.0" />

</Project>
```

Run the following command:

```shell
dotnet publish SimpleAOT\SimpleAOT.csproj
```

The result is:

```
MSBuild version 17.6.0-preview-23179-01+dd5d9f746 for .NET
  Determining projects to restore...
  Restored C:\Users\rodrigov\source\repos\SimpleAOT\SimpleAOT\SimpleAOT.csproj (in 167 ms).
C:\Program Files\dotnet\sdk\7.0.300-preview.23179.2\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.RuntimeIdentifierInfer
ence.targets(287,5): message NETSDK1057: You are using a preview version of .NET. See: https://aka.ms/dotnet-support-po
licy [C:\Users\rodrigov\source\repos\SimpleAOT\SimpleAOT\SimpleAOT.csproj]
  SimpleAOT -> C:\Users\rodrigov\source\repos\SimpleAOT\___output\Debug\AnyCPU\SimpleAOT\net7.0\win-x64\SimpleAOT.dll
  Generating native code
     Creating library C:\Users\rodrigov\source\repos\SimpleAOT\___publish\Debug\AnyCPU\SimpleAOT\native\SimpleAOT.lib a
  nd object C:\Users\rodrigov\source\repos\SimpleAOT\___publish\Debug\AnyCPU\SimpleAOT\native\SimpleAOT.exp
C:\Program Files\dotnet\sdk\7.0.300-preview.23179.2\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Publish.targets(267,5)
: error MSB3030: Could not copy the file "C:\Users\rodrigov\source\repos\SimpleAOT\___publish/Debug/AnyCPU/SimpleAOT/ne
t7.0/win-x64/native/SimpleAOT.exe" because it was not found. [C:\Users\rodrigov\source\repos\SimpleAOT\SimpleAOT\Simple
AOT.csproj]
```

--- a/src/CentralBuildOutput.Tests/CentralBuildOutputTests.cs
+++ b/src/CentralBuildOutput.Tests/CentralBuildOutputTests.cs
@@ -186,7 +186,6 @@ public class CentralBuildOutputTests : MSBuildSdkTestBase
 
         CommonMSBuildMacros msbuildMacros = properties.MSBuildMacros;
         msbuildMacros.PublishDir.MakeRelative(this.ProjectOutput).ShouldBe("__publish/Debug/AnyCPU/src/MyClassLibrary/");
-        msbuildMacros.NativeOutputPath.MakeRelative(this.ProjectOutput).ShouldBe("__publish/Debug/AnyCPU/src/MyClassLibrary/native/");
 
         MSBuildOtherProperties msBuildOtherProps = properties.MSBuildOther;
         msBuildOtherProps.MSBuildProjectExtensionPath.MakeRelative(this.ProjectOutput).ShouldBe("__intermediate/src/MyClassLibrary/");
@@ -793,7 +792,6 @@ public class CentralBuildOutputTests : MSBuildSdkTestBase
 
         CommonMSBuildMacros msbuildMacros = properties.MSBuildMacros;
         msbuildMacros.PublishDir.MakeRelative(this.ProjectOutput).ShouldBe("__publish/Debug/AnyCPU/src/MyClassLibrary/win10-x64/");
-        msbuildMacros.NativeOutputPath.MakeRelative(this.ProjectOutput).ShouldBe("__publish/Debug/AnyCPU/src/MyClassLibrary/win10-x64/native/");
 
         MSBuildOtherProperties msBuildOtherProps = properties.MSBuildOther;
         msBuildOtherProps.MSBuildProjectExtensionPath.MakeRelative(this.ProjectOutput).ShouldBe("__intermediate/src/MyClassLibrary/");

--- a/src/CentralBuildOutput/Sdk/ConfigureBuildOutput.targets
+++ b/src/CentralBuildOutput/Sdk/ConfigureBuildOutput.targets
@@ -137,7 +137,6 @@
     <BaseOutputPath>$(BaseProjectOutputPath)</BaseOutputPath>
     <OutputPath>$(BaseProjectOutputPath)</OutputPath>
     <PublishDir>$(BaseProjectPublishOutputPath)</PublishDir>
-    <NativeOutputPath>$(PublishDir)native/</NativeOutputPath>
     <VSTestResultsDirectory>$(BaseProjectTestResultsOutputPath)</VSTestResultsDirectory>
     <CoverletOutput>$(BaseProjectTestResultsOutputPath)</CoverletOutput>
   </PropertyGroup>
@@ -175,7 +174,6 @@
     -->
     <PropertyGroup Condition=" '$(CentralBuildOutputDotNetStyleProject)' == 'true' ">
       <PublishDir>$(BaseProjectPublishOutputPath)</PublishDir>
-      <NativeOutputPath>$(PublishDir)native/</NativeOutputPath>
     </PropertyGroup>
 
   </Target>

--- a/version.json
+++ b/version.json
@@ -1,6 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/AArnott/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "2.0",
+  "version": "3.0",
   "buildNumberOffset": -1,
   "nugetPackageVersion": {
     "semVer": 2

